src/components/App.js

```diff
@@ -1,30 +1,24 @@
-/* eslint-disable no-unused-vars */
 import { useState } from 'react';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
-import Calculator from '../logic/calculate';
+import calculator from '../logic/calculate';
 
 function App() {
-  const [value, setValue] = useState({ total: '0' });
+  const [value, setValue] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
   const handleClick = (name) => {
-    let newValue;
-    if (value.total === '0') {
-      newValue = '';
-    } else {
-      newValue = value.total;
-    }
-
-    if (name === 'AC') {
-      setValue(value.total = { total: '0' });
-    } else {
-      newValue += name;
```

Comment:
- [ ] Convert the `App` component to a stateful component using the __class__ syntax. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))
- [ ] Have the state initialized in constructor ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))